README.md

```diff
@@ -5,37 +5,50 @@
 
 Unleash your creativity with presentations like never before! FlutterShow⚡️ is an easy-to-use framework built in Flutter for crafting engaging and interactive presentations.
 
-Check out the full documentation showcasing all slides here: https://flutter-show-docs.vercel.app/#/
+Check out the full documentation showcasing all slides and code snippets to speed up your development time here: https://flutter-show-docs.vercel.app/#/
 
 ![Showcase](docs/images/showcase.png)
 
 ## General Information
 
 FlutterShow consists of two main packages: `fluttershow_base` and `fluttershow_keynote`:
 
-- [fluttershow_base](https://pub.dev/packages/fluttershow_base) contains most of the pre-built widgets that are used in `fluttershow_keynote`, but can also be used to make custom slides.
-- [fluttershow_keynote](https://pub.dev/packages/fluttershow_keynote) is currently the default package and implements all basic slides with variations of the popular presentation software Keynote.
+- [fluttershow_base](https://github.com/lucas-goldner/fluttershow_base) contains most of the pre-built widgets that are used in `fluttershow_keynote`, but can also be used to make custom slides.
+- [fluttershow_keynote](https://github.com/lucas-goldner/fluttershow_keynote) is currently the default package and implements all basic slides with variations of the popular presentation software Keynote.
```

Comment:
I feel the link here(both packages) should remain pub.dev. And then one can go to the GitHub repository from pub page.

Reply:
I felt the same 🤣 , but @dumazy initially did not find them, so I did the reverse. I added the repos but linked put the pub.dev links in their respective repos. @dumazy What do you think now about this topic?

Reply:
I was okay that the links here went to pub.dev, but on pub.dev the repositories for the separate packages pointed back to `fluttershow` here, not to their own source code, so I couldn't find where to contribute to `fluttershow_base` or `fluttershow_keynote` :D

Reply:
I noticed this issue today as well, so that also needs to be fixed ^^